### PR TITLE
[OPENY-240] Breadcrumbs aren't visible

### PR DESCRIPTION
**Pull request by miroslav-lee**
## Issue details:
After adding Limited Time Offer paragraph, we do not see a block with breadcrumbs
not visible

## Step for review:
- [x] Log in as admin
- [x] Add new page of Landing page CT
- [x] Add Limited Time Offer paragraph to Content area
- [x] Save this page
- [x] Check that breadcrumbs are visible.
breadcrumbs

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: themes/openy_themes/openy_rose/css/styles.css [hunk 1]
```diff
@@ -8094,7 +8094,6 @@ article .node--view-mode-flexible-content h2 a span {
 .paragraph--type--lto {
   background-color: #0060af;
   padding: 20px 0;
-  margin: -20px 0 40px;
```
Comment (hamrant) [inline themes/openy_themes/openy_rose/css/styles.css:8097]: I think we need some positive margin top and bottom in this paragraph.

**Review by hamrant — CHANGES_REQUESTED** [review]

File: themes/openy_themes/openy_rose/css/styles.css (hunk 1 above)
Comment (hamrant) [reply themes/openy_themes/openy_rose/css/styles.css:8097]: Let's use `margin: 20px 0;` or `margin: 40px 0;`

Comment (miroslav-lee) [reply themes/openy_themes/openy_rose/css/styles.css:8097]: @hamrant I fixed it

margin

**Review by ddrozdik — APPROVED** [review]

**Comment by ddrozdik** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1256 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1256 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1256 |
| Installation process | http://install.openy.cibox.tools/build1256/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1256/

**Comment by ddrozdik** [comment]
retest this please

**Review by hamrant — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation - Rose theme | http://openy.cibox.tools/build1270 |
| Fresh OpenY installation - Lily theme | http://lily.openy.cibox.tools/build1270 |
| Upgraded(upgrade path) installation | http://upgrade.openy.cibox.tools/build1270 |
| Installation process | http://install.openy.cibox.tools/build1270/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1270/

**Review by podarok — APPROVED** [review]

**Pull request merged**